src/azure-cli/azure/cli/command_modules/redis/tests/latest/test_redis_scenario.py

```diff
@@ -32,7 +31,7 @@ def test_redis_cache_with_mandatory_args(self, resource_group):
         self.cmd('az redis create -n {name} -g {rg} -l {location} --sku {sku} --vm-size {size}')
         self.cmd('az redis show -n {name} -g {rg}', checks=[
             self.check('name', '{name}'),
-            self.check('provisioningState', 'Creating'),
+            self.check('provisioningState', 'Succeeded'),
```

Comment:
May I ask why `provisioningState` has changed?

Reply:
We have implemented async header in this new api version.
That could have caused it.